CONTRIBUTING.md

```diff
@@ -89,6 +89,36 @@ After build completes, you can preview the resulting site locally using built-in
 	
 This will start the local preview on localhost:8080. You can then view the changes by going to http://localhost:8080/docs/[path] (e.g. http://localhost:8080/docs/getting-started/).   
 
-## Contributing to samples
+# Contributing to samples
 
-See the [Samples Readme](https://github.com/dotnet/core-docs/blob/master/samples/README.md).
+We encourage samples for any of our topics. Readers can download and run the samples. They
+can explore the code and experiment with their own changes.
+
+All samples should be complete applications or libraries. Where the sample creates a library,
+it should include unit tests or an application that lets readers run the code.
+
+The samples are organized in directories that match the directories under
+docs in this repo:
+
+- `core-projects` contains all .NET Core samples. These are referenced by topics under `/docs/core`.
+
+- `charp-language` contains all C# language samples. These are referenced by topics under `docs/csharp`.
+
+- `getting-started` contains all getting started samples. These are referenced by topics under `docs/*/tutorials`
+
+- `linq` contains all the 101 LINQ samples that were originally created when LINQ was developed. These don't have topics associated with them yet.
+
+NOTE to Reviewers: The C# Tutorials (/docs/csharp/tutorials) have their sample code in /samples/csharp. I think that
```

Comment:
"contains all C#" or "contains all Core" statements are misleading. A given sample could fit in all of the first three categories; could give guidance on how to pick which one applies:

1. getting started -- all getting started for core or csharp
2. core-projects -- main purpose of sample is to demo .NET core features
3. csharp -- main purpose of sample is to demo csharp features